### PR TITLE
Finished codebreacker rack

### PR DESCRIPTION
**Pull request by bezrukavyi**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+source 'https://rubygems.org'
+
+gem 'codeguessing'
+gem 'rack'
+gem 'json'
+gem 'pry'
+
+group :development, :test do
```
Comment (Alfarlost) [inline Gemfile]: jsut, a little question, why do you need rspec/capybara in test ENV

Comment (bezrukavyi) [inline Gemfile]: I took this code from rails app and forgot remove excess

**Comment by bezrukavyi** [comment]
Круто! Не ожидал, спасибо

**Comment by Alfarlost** [comment]
np, just for practice, add some kind of authorization. if you want, ofc

**Comment by bezrukavyi** [comment]
Standard authorization with email and password?
Okay, it will be interesting.

**Pull request merged**